[TECH] Casser les dépendances de librairie entre racine et application

yannbertrand opened the pull request:
## :unicorn: Problème
La CI est longue, et les tests de chaque application sont dépendant de l'installation de la racine à cause de dépendances non présentes dans les applications (mise en commun de code):
- `npm-run-all` pour exécuter la CLI `run-p`.
- `eslint-plugin-i18n-json` (qui vient de `@1024pix/eslint-config`) pour linter les fichiers de traduction.

Voir PR  #6976

## :robot: Proposition
Rapatrier toutes les dépendances dans les applications pour qu'elles soient indépendantes.

## :rainbow: Remarques
La parallélisation des tests de la racine et de chaque application sera faite une PR suivante.

## :100: Pour tester
En local, supprimer les `nodes_modules` à la racine et s'assurer que les sous-dossiers fonctionnent après un `npm ci` (`lint`, `test`, `start`...)

Sur la CI, qu'elle passe.

yannbertrand committed:
chore(admin): add missing dependencies

yannbertrand committed:
chore(1d): add missing dependencies

yannbertrand committed:
chore(certif): add missing dependencies

yannbertrand committed:
chore(orga): add missing dependencies

pix-bot-github commented:
Une fois les applications déployées, elles seront accessibles via les liens suivants :
  * [App (.fr)](https://app-pr6978.review.pix.fr)
  * [App (.org)](https://app-pr6978.review.pix.org)
  * [Orga (.fr)](https://orga-pr6978.review.pix.fr)
  * [Orga (.org)](https://orga-pr6978.review.pix.org)
  * [Certif (.fr)](https://certif-pr6978.review.pix.fr)
  * [Certif (.org)](https://certif-pr6978.review.pix.org)
  * [1D](https://1d-pr6978.review.pix.fr)
  * [Admin](https://admin-pr6978.review.pix.fr)
  * [API](https://api-pr6978.review.pix.fr/api/)
  * [Audit Logger](https://pix-audit-logger-review-pr6978.osc-fr1.scalingo.io/api/)

Les variables d'environnement seront accessibles via les liens suivants :
  * [scalingo front](https://dashboard.scalingo.com/apps/osc-fr1/pix-front-review-pr6978/environment)
  * [scalingo api](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-review-pr6978/environment)
  * [scalingo audit-logger](https://dashboard.scalingo.com/apps/osc-fr1/pix-audit-logger-review-pr6978/environment)

yannbertrand committed:
chore(audit-logger): add @1024pix/eslint-config dependency locally

It was in fact already used because of ESLint's cascading strategy, https://eslint.org/docs/latest/use/configure/configuration-files\#cascading-and-hierarchy

octo-topi reviewed (approved):
C'est tellement bien de casser des dépendances, merci Yann !
J'ai modifié la descritption de PR.

yannbertrand committed:
chore(api): add missing dependencies

yannbertrand committed:
chore(mon-pix): add missing dependencies
